lint/auth_account_parameter_analyzer_test.go

```diff
@@ -32,10 +32,13 @@ func TestAuthAccountParameterAnalyzerInContractFunction(t *testing.T) {
 
 	t.Parallel()
 
+	// TODO: No longer valid?
+	t.SkipNow()
```

Comment:
The analyzer should report uses of `auth(...) &Account` now.

Maybe open a follow-up issue for that

Reply:
Opened https://github.com/onflow/cadence-tools/issues/212